docs/UserGuide.adoc

```diff
@@ -396,3 +423,14 @@ e.g. `warning e/20 s/10`
 * *Back to input* : kbd:[Shift] + kbd:[Enter]
 * *To bottom of list* : kbd:[Shift] + kbd:[d]
 * *To top of list* : kbd:[Shift] + kbd:[u]
+
+== Appendix A: File Naming Convention
+
+MediTabs uses the following File Naming Convention when File Name field is used e.g. The `export` command's optional `[FILE_NAME]` field. The File Naming Convention is to ensure consistency and to avoid potential bugs involving file names with different operating systems such as Windows, Linux and MacOS +
+Format: `Start with an Alphabet or Number followed by Alphabets, Numbers, underscore or hyphen`
+
```

Comment:
not sure if this should be in single quotes since it is not code. Also Alphabets and Numbers do not need to be caps?

Reply:
It is indeed not a code but a format and I wrote it as based on consistency with the rest of the “format:” I was thinking of writing alphanumeric etc but I think when we discussed a few weeks ago you stated we do not have to write that as some might not know what alphanumeric characters are. Maybe I can add that as a terminology in future commits